src/PowerShellEditorServices/Services/DebugAdapter/Handlers/LaunchAndAttachHandler.cs

```diff
@@ -128,6 +139,20 @@ public LaunchAndAttachHandler(
         }
 
         public async Task<LaunchResponse> Handle(PsesLaunchRequestArguments request, CancellationToken cancellationToken)
+        {
+            _debugService.SetPathMappings(request.PathMappings);
+            try
+            {
+                return await HandleImpl(request, cancellationToken).ConfigureAwait(false);
+            }
+            catch
+            {
+                _debugService.UnsetPathMappings();
+                throw;
```

Comment:
I feel like there's a better pattern to apply here, but I'm not sure what. @SeeminglyScience for a suggestion (is there a way this can be done with a `using`? Is that too complicated?)

Reply:
@jborean93 should this be a `finally` instead of a `catch`? or are the path mappings unset somewhere else when there's no error? I guess that would have to be the case right? `Handle` returns before debugging is actually complete right?

(if the answer to the above is "no it needs to be a catch", then @andyleejordan: nah `using` wouldn't work here unfortunately as it's only unset when an exception occurs)